nemo/collections/nlp/data/machine_translation/preproc_mt_data.py

```diff
@@ -829,8 +829,7 @@ def train_tokenizers(
                                     split_by_whitespace=split_by_whitespace,
                                 )
                                 os.rename(
-                                    os.path.join(out_dir, 'tokenizer.model'),
-                                    os.path.join(out_dir, encoder_tokenizer_model),
+                                    os.path.join(out_dir, 'tokenizer.model'), os.path.join(encoder_tokenizer_model),
```

Comment:
Can we omit the second `os.path.join`?

```suggestion
                                    os.path.join(out_dir, 'tokenizer.model'), encoder_tokenizer_model,
```

Reply:
Removed all superfluous calls of `os.path.join` in this file.